install_files/ansible-base/roles/app/tasks/initialize_securedrop_app.yml

```diff
@@ -141,6 +141,14 @@
     - gpg
     - securedrop_config
 
+- name: Copy custom notification text to config.py.
+  lineinfile:
+     dest: "{{ securedrop_code }}/config.py"
+     regexp: "^CUSTOM_NOTIFICATION"
+     line: "CUSTOM_NOTIFICATION = '{{ custom_notification }}'"
```

Comment:
1. Var should be referred to as `custom_notification_text` for consistency.
2. Tack on a conditional to ensure Ansible responds sanely to presence or absence of this var:
```
when: custom_notification_text is defined and custom_notification_text != ''
```